musicbrainz/csv/get_genre.py

```diff
@@ -0,0 +1,64 @@
+"""
+Script for retrieving the "genre" dumps since they are not available online.
```

Comment:
```suggestion
Script for retrieving the "genre" dumps.
```
They are available online...that's how you are getting them in this script.